Gemfile

```diff
@@ -18,10 +18,9 @@ gem 'kaminari', '~> 1.1.1'
 gem "simple_form", ">= 3.0.0"
 gem 'rollbar', '2.8.3'
 gem 'whenever', require: false
+gem 'pg_search', '2.1.4'
 gem 'prawn', '~> 2.2.0'
 gem 'prawn-table', '~> 0.2.2'
-gem 'elasticsearch-model'
-gem 'elasticsearch-rails'
```

Comment:
This change shouldn't be here

Reply:
I restored the GemFile from https://github.com/coopdevs/timeoverflow/blob/develop/Gemfile, those lines weren't there. Should it be present @enricostano ?

Reply:
You shouldn't need to touch `Gemfile`. Just use the one that you have in `develop`. Is you fork up to date with upstream?

https://help.github.com/en/articles/syncing-a-fork

Reply:
It wasn't up to date. Now it is, but while starting timeoveflow server

`bundle exec rails s --binding 0.0.0.0`

And error appears: 

timeoverflow@timeoverflow:/var/www/timeoverflow$ bundle exec rails s --binding 0.0.0.0
=> Booting WEBrick
=> Rails 4.2.11 application starting in development on http://0.0.0.0:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
W, [2019-06-21T11:58:59.836996 #1339]  WARN -- Skylight: [SKYLIGHT] [2.0.1] Running Skylight in development mode. No data will be reported until you deploy your app.
(To disable this message for all local apps, run `skylight disable_dev_warning`.)
Exiting
/var/www/timeoverflow/app/models/post.rb:3:in `<class:Post>': uninitialized constant Post::PgSearch (NameError)
	from /var/www/timeoverflow/app/models/post.rb:1:in `<top (required)>'
	from /var/www/timeoverflow/app/admin/post.rb:1:in `<top (required)>'


Did I miss a new step before starting? @enricostano

Reply:
`bundle install` ?

Reply:
No, `bundle install` finished ok

Reply:
Done, it works now, thank's for your help @sseerrggii!